src/clj/ataru/forms/form_payment_info.clj

```diff
@@ -72,7 +75,7 @@
   "Adds the payment type info to a form if valid"
   [form payment-type]
   (if (contains? form-payment-types payment-type)
-    (assoc-in form [:properties :payment-type] payment-type)
+    (assoc-in form [:properties :payment-type] (name payment-type))
```

Comment:
Kaikki propertyjen arvot näkyvät kulkevan formissa koodissakin stringeinä keywordien sijaan, joten olkoon tämäkin yhdenmukaisuuden nimissä näin.